pyomo/util/subsystems.py

```diff
@@ -75,7 +111,12 @@ def add_local_external_functions(block):
     return fcn_comp_map
 
 
-def create_subsystem_block(constraints, variables=None, include_fixed=False):
+from pyomo.common.timing import HierarchicalTimer
```

Comment:
Can this import be moved to the top of the file?

Reply:
I will decide what (if any) timing calls I want to keep in these helper functions, then move (or remove) this import.